Exceptions&Debugging.py

```diff
@@ -0,0 +1,18 @@
+#EXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUGEXCEPTION&DEBUG
+# 1.
+try:
+    div_zero = int(input())
+    div_zero_act = div_zero/0
+    print(div_zero)
+except:
```

Comment:
```suggestion
except ZeroDivisionError:
```
You should always catch exact error that may appear